vignettes/wrapping-apis.Rmd

```diff
@@ -698,13 +734,15 @@ You've almost certainly used this before on the web; it's used in most cases whe
 
 ### Clients
 
-The first step in working with any OAuth api is to create a client.
-This involves you registering for a developer account on the APIs website and creating a new OAuth app.
+The first step in working with any OAuth API is to create a client.
+This involves you registering for a developer account on the API's website and creating a new OAuth app.
 The process varies from API to API, but at the end of it you'll get a client id and in most cases a client secret.
 
+JENNY COMMENT HERE
```

Comment:
I feel like the scenario of "user provides own client" is not on the radar here at all, but it should be.